frontend/ui/elements/screen_fullscreen_menu_table.lua

```diff
@@ -0,0 +1,19 @@
+local InfoMessage = require("ui/widget/infomessage")
+local UIManager = require("ui/uimanager")
+local _ = require("gettext")
+local android = require("device/android/device")
+
+return {
+    text = _("Fullscreen"),
+    checked_func = function()
+        return G_reader_settings:readSetting("disable_fullscreen") ~= false
+    end,
+    callback = function()
+        local disabled = G_reader_settings:readSetting("disable_fullscreen") ~= false
+        G_reader_settings:saveSetting("disable_fullscreen", not disabled)
+        android.setFullscreen(disabled)
```

Comment:
Does it make a difference if you use `:` instead of `.`? Looking at the whole thing here rather than the sample you posted earlier I realized we were talking about the `Device` method rather than a direct reference. You might have to pass in a reference to the object if you do it this say. That'd be the less elegant `android.setFullscreen(android, disabled)`.[1] In that case it'd think `disabled` is the object reference and fullscreen `nil` if you only pass android.setFullscreen(disabled).

[1] It'd also be the wrong `android` object as it's actually from `Device`.